### PR TITLE
added report command + fixed skill issue

### PR DESCRIPTION
**Pull request by qexat**

### CODE REVIEW

File: src/logic/logic.shaii.ts [hunk 1]
```diff
@@ -40,19 +40,26 @@ export function timeSince(date: number) {
 export function durationToMilliseconds(duration: string): string {
   const [digits, abbr] = duration.match(/\d{1,3}|\D/g) as string[];
   switch (abbr) {
-    case "s" || "S":
+    case "s":
```
Comment (Geoxor) [inline src/logic/logic.shaii.ts:43]: why was this changed

**Review by Geoxor — APPROVED** [review]

File: src/logic/logic.shaii.ts (hunk 1 above)
Comment (qexat) [reply src/logic/logic.shaii.ts:43]: Because it's not Python it does not work that way xd
